jobs/rocoto/oceanice_products.sh

```diff
@@ -15,11 +15,17 @@ if (( status != 0 )); then exit "${status}"; fi
 export job="oceanice_products"
 export jobid="${job}.$$"
 
-export FORECAST_HOUR=$(( 10#${FHR3} ))
+# shellcheck disable=SC2153
+IFS=', ' read -r -a fhr_list <<< "${FHR_LIST}"
 
-###############################################################
-# Execute the JJOB
-###############################################################
-"${HOMEgfs}/jobs/JGLOBAL_OCEANICE_PRODUCTS"
+export FORECAST_HOUR
+for FORECAST_HOUR in "${fhr_list[@]}"; do
+	###############################################################
+	# Execute the JJOB
+	###############################################################
+	"${HOMEgfs}/jobs/JGLOBAL_OCEANICE_PRODUCTS"
```

Comment:
@EricSinsky-NOAA's suggestion for jobid should be done here as well.  In addition, since rocoto sets `COMPONENT`, may be we can use that as well in `jobid` such that `${job}_${COMPONENT}_${FORECAST_HOUR}.$$`
```suggestion
	"${HOMEgfs}/jobs/JGLOBAL_OCEANICE_PRODUCTS"
	jobid="${job}_${COMPONENT}_${FORECAST_HOUR}.$$"
```